srcpkgs/dbeaver/files/dbeaver.desktop

```diff
@@ -1,9 +1,8 @@
 [Desktop Entry]
 Name=DBeaver
 Comment=SQL Integrated Development Environment
-Icon=dbeaver
+Icon=/usr/lib/dbeaver/icon.xpm
```

Comment:
@ar-jan why did you change the icon path?
this will break other icon like `papirus-icons` which already provide default icon from the theme.

Reply:
`Icon=dbeaver` never worked for me so I had no icon for dbeaver (I use KDE5 Plasma). I wasn't aware this change would break other configs, how does `papirus-icons` work?

Reply:
didn't work, `papirus-icon-theme` couldn't read it.